main.js

```diff
@@ -1,103 +1,117 @@
-createLoginForm();
+let validationErrors = {};
+let formValues = {};
+let touchedElements = [];
 
 const userInfo = {
     username: "user",
     password: "test",
     token: ""
 };
 
-const loginForm = document.querySelector("#login")
+createLoginForm();
+
+const loginForm = document.querySelector("#login");
 const usernameInput = document.querySelector("#username");
 const userPassword = document.querySelector("#password");
+const submitButton = document.querySelector("#login_btn");
+
+document.addEventListener("input", function(){
+    setFormValue("valueUsername",usernameInput.value)
+    setFormValue("valuePassword",userPassword.value)
+});
+
+loginForm.addEventListener("click", function(e) {
```

Comment:
Move `addEventListener()` inside `createInput()` and `createButton()`

Reply:
Done